app/models/peer_review.rb

```diff
@@ -69,16 +69,61 @@ def self.assign(reviewer_groups, reviewee_groups)
   end
 
   def self.unassign(selected_reviewee_group_ids, reviewers_to_remove_from_reviewees_map)
+    delete_all_reviews = true
+    deleted_count = 0
+    undeleted_reviews = []
+
+    # When an entire row is selected for unassignment
```

Comment:
This is interesting, it's seems you're actually fixing a separate bug here; please pull this out and make into into a new pull request with a description of just this bug.